python-checks/src/main/java/org/sonar/python/checks/DeadStoreCheck.java

```diff
@@ -113,12 +117,41 @@ private static void raiseIssue(SubscriptionContext ctx, DeadStoreUtils.Unnecessa
       .forEach(tree -> issue.secondary(tree, String.format(SECONDARY_MESSAGE_TEMPLATE, symbolName)));
 
     if (element instanceof Statement && !isExceptionForQuickFix((Statement) element)) {
-      ((IssueWithQuickFix) issue).addQuickFix(PythonQuickFix.newQuickFix("Remove the unused statement",
-        PythonTextEdit.removeStatement((Statement) element)));
+      if (element.is(Tree.Kind.ASSIGNMENT_STMT) && ((AssignmentStatement) element).lhsExpressions().size() > 1) {
+        addMultipleAssignmentStatementQuickFix((AssignmentStatement) element, issue, unnecessaryAssignment.symbol);
+      } else {
+        issue.addQuickFix(PythonQuickFix.newQuickFix(QUICK_FIX_MESSAGE,
+          PythonTextEdit.removeStatement((Statement) element)));
+      }
     }
 
   }
 
+  private static void addMultipleAssignmentStatementQuickFix(AssignmentStatement element, IssueWithQuickFix issue, Symbol symbol) {
+    var children = element.children();
+
+    IntStream.range(0, children.size())
+      .filter(i -> isExpressionHasSymbol(children.get(i), symbol))
+      .findFirst()
+      .ifPresent(i -> {
+        var from = i == 0 ? i : (i - 1);
+        var to = i == 0 ? (i + 2) : (i + 1);
```

Comment:
I think that another way to write it would be:
```
var to = from + 2;
```
Not sure which one is better.

Reply:
good catch, done